src/rest-server/src/middlewares/v2/hived.js

```diff
@@ -216,16 +217,24 @@ const hivedValidate = async (protocolObj, username) => {
       };
     }
 
-    if (gpu > resourcePerCell.gpu * cellNumber ||
-        cpu > resourcePerCell.cpu * cellNumber ||
-        memoryMB > resourcePerCell.memory * cellNumber) {
+    const {gpu = 0, cpu, memoryMB} = protocolObj.taskRoles[taskRole].resourcePerInstance;
+    if (resourcePerCell.gpu === 0 && gpu > 0 ||
+        resourcePerCell.cpu === 0 && cpu > 0 ||
+        resourcePerCell.memory === 0 && memoryMB > 0) {
       throw createError(
         'Bad Request',
         'InvalidProtocolError',
-        `Taskrole ${taskRole} requests ${gpu} GPU, ${cpu} CPU, ${memoryMB}MB memory; ` +
-        `sku allows ${resourcePerCell.gpu} GPU, ${resourcePerCell.cpu} CPU, ${resourcePerCell.memory}MB memory per cell.`
+        `Taskrole ${taskRole} requests (${gpu} GPU, ${cpu} CPU, ${memoryMB}MB memory), ` +
```

Comment:
"Minimum SKU" limitation is confusing. May we create a doc to introduce why we introduce "minimum sku" limit for users reference?

Reply:
removed

Reply:
@abuccts - after removed that limit, the sentence seems not providing any actions to the user? we shall provide an actionable follow up to the user, so that they can fix the error by themselves.

Reply:
`Taskrole worker requests (1 GPU, 4 GPU, 4096MB memory), but SKU does not configure GPU` means user couldn't request GPU, the config file should contain 0 GPU.

Reply:
Taskrole worker requests 1 _GPU_, but there are SKU does not configure _GPU_. Please contact admin if the taskrole need _GPU_ resources.